### ISSUE TITLE
aria-disabled is not passed to MenuItem/ListBoxItem/Tab/TreeItem

### ISSUE THREAD

**Issue by phuoccss** [adobe/react-spectrum#10276]
### Provide a general summary of the feature here

I'm trying to implement focusable, disabled MenuItem (https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#focusabilityofdisabledcontrols) but it is not passed to DOM element. ListBoxItem and Tab also have this issue. Haven't checked TreeItem.

### 🤔 Expected Behavior?

MenuItem div element should have aria-disabled.

### 😯 Current Behavior

aria-disabled is not passed, unlike Button.

### 🔦 Context

https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#focusabilityofdisabledcontrols
